frontend/wallet/src/main/ProjectRoot.re

```diff
@@ -32,12 +30,12 @@ let userData =
   if (isPackaged) {
     getPath(`UserData);
   } else {
-    Node_path.join2(Option.getExn([%bs.node __dirname]), "../../../..");
+    Node.Process.cwd();
   };
 
 let resource =
   if (isPackaged) {
     Filename.dirname(getAppPath());
   } else {
-    Node_path.join2(Option.getExn([%bs.node __dirname]), "../../../..");
+    Node.Process.cwd();
```

Comment:
This isn't going to work when I'm loading this directly from the render process right?

Reply:
Mhh, not sure, you would only be loading from the section of the render process that has access to the node apis, right? Not totally sure how that works